src/assets/styles/_globals.scss

```diff
@@ -0,0 +1,9 @@
+//
+// Warning: Make sure this manifest never outputs CSS
+// and only import variables, mixins, functions etc.
+// as this file is included in _every_ component
```

Comment:
I took a slightly different approach on CEVT where I only included relevant dependencies in each component rather than all of them. Slightly more work but also more explicit and synced with the JS way of doing.

E.g.: https://github.com/edenspiekermann/cevt/blob/master/assets/sass/components/_button.scss#L1-L2.